_components/modal/guidance/implementation.md

```diff
@@ -1,6 +1,6 @@
 - **Use unique ids.** Each `.usa-modal` must have a unique id so that openers can associate them with their `aria-controls` attribute.
 
-- **Openers.** A single modal can have multiple openers. Each opener requires `data-open-modal` and `aria-controls=”MODAL_ID”` attributes. Openers can be coded either as `<button type="button">` or `<a>`. Using `<a>` helps link to modals in the event javascript fails.
```

Comment:
> **Note** 
> Removed this because it doesn't seem like there is a progressive enhancement here. I might have misunderstood the meaning though, so happy to return if it is still relevant.